.github/workflows/wolfictl-update-gh.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
     steps:
     - uses: wolfi-dev/wolfictl/.github/actions@main
     - run: |
-        git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+        git config --global user.email "121097084+wolfi-bot@users.noreply.github.com"
         git config --global user.name "github-actions[bot]"
```

Comment:
maybe change this name as well? `wolfi-bot` ?

Reply:
good spot!